README.md

```diff
@@ -0,0 +1,61 @@
+# Conduit Connector NATS JetStream
+
+### General
+
+The [NATS](https://nats.io/) JetStream connector is one of [Conduit](https://github.com/ConduitIO/conduit) plugins. It provides both, a source and a destination NATS JetStream connector.
+
+### Prerequisites
+
+- [Go](https://go.dev/) 1.18
+- (optional) [golangci-lint](https://github.com/golangci/golangci-lint) 1.45.2
+- [NATS](https://nats.io/download/) 2.8.4
+- [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/).
+
+### How to build it
+
+Run `make`.
+
+### Testing
+
+Run `make test` to run all the unit and integration tests, which require Docker and Docker Compose to be installed and running. The command will handle starting and stopping docker containers for you.
+
+## Source
+
+### Connection and authentication
+
+The NATS JetStream connector connects to a NATS server or a cluster with the required parameters `urls`, `subject` and `mode`. If your NATS server has a configured authentication you can pass an authentication details in the connection URL. For example, for a token authentication the url will look like: `nats://mytoken@127.0.0.1:4222`, and for a username/password authentication: `nats://username:password@127.0.0.1:4222`. But if your server is using [NKey](https://docs.nats.io/using-nats/developer/connecting/nkey) or [Credentials file](https://docs.nats.io/using-nats/developer/connecting/creds) for authentication you must configure them via separate [configuration](#configuration) parameters.
+
+### Receiving messages
+
+The connector creates a durable NATS consumer which means it's able to read messages that were written to a NATS stream before the connector was created, unless configured otherwise. The `deliverPolicy` configuration parameter allows you to control this behavior.
+
+- If the `deliverPolicy` is equal to `new` the connector will only consume messages which were created after the connector.
```

Comment:
> which were created after the connector.

I believe we should add `after the connector started`, to make it clear.

Reply:
Fixed in the https://github.com/conduitio-labs/conduit-connector-nats-jetstream/pull/5/commits/3c798bf55b81ba4077ef5bf0460b262f23162811